### ISSUE TITLE
Shear operation

### ISSUE THREAD

**Issue by Zireael07** [disintegration/imaging#120]
A combination of scale, resize and shear can be used to convert an image to isometric perspective.

**Comment by disintegration** [comment]
Indeed, the shear operation would be useful in some cases. Contributions are welcome!
